src/thing_doors.c

```diff
@@ -311,6 +311,29 @@ TbBool thing_is_deployed_door(const struct Thing *thing)
     return true;
 }
 
+TbBool thing_is_sellable_door(const struct Thing* thing)
+{
+    if (thing_is_invalid(thing))
+        return false;
+    if (thing->class_id != TCls_Door)
+        return false;
+    struct DoorConfigStats* doorst = &gameadd.trapdoor_conf.door_cfgstats[thing->model];
```

Comment:
This would be nicer as:

`struct DoorConfigStats* doorst = get_door_model_stats(thing->model)`